src/hooks/useOAuthFlow.tsx

```diff
@@ -83,6 +93,10 @@ export const OAuthContextProvider = ({
     [authConfig, clearAuthResult, storeAuthResult],
   );
 
+  useEffect(() => {
+    initialize();
+  }, [initialize]);
+
```

Comment:
Could this just be moved into `useAuth`?

Reply:
The `initialize` call?  It could, but it creates some testing challenges.  When you have a no-deps, first-render useEffect, it makes it more challenging to test.  In this repo, I was running into errors about that first render not being in an `act` but having a side effect.  I could see there being some really nice advantages to externalizing that initialization (perhaps optionally, long-term).  I do agree it feels weird in here - or will long-term.. let's definitely keep playing with this as we go.